cmd/restartComponent.go

```diff
@@ -0,0 +1,75 @@
+// Copyright © 2019 NAME HERE <EMAIL ADDRESS>
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package cmd
+
+import (
+	"errors"
+	"fmt"
+
+	"github.com/equinor/radix-cli/generated-client/client/component"
+	"github.com/equinor/radix-cli/pkg/client"
+	"github.com/spf13/cobra"
+)
+
+const restartComponentEnabled = true
+
+// restartComponentCmd represents the restart component command
+var restartComponentCmd = &cobra.Command{
+	Use:   "component",
+	Short: "Restart a component",
+	Long:  `Redeploys and starts a component for the application`,
```

Comment:
Have to write a long text that explains what happens during Restart.
- Stops running container 
- Pull image specified for component from image hub
- Starts container

Not sure how this can be written in a good way. Maybe it should just be linked to in our documentation? Any suggestion?